### PR TITLE
feat: node module resolution compiler option in backend

### PR DESCRIPTION
**Pull request by inpaceee**
closes: #947

### CODE REVIEW

**Review by YohannesTesfay — DISMISSED** [review]

File: packages/backend/tsconfig.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
   "include": ["src"],
   "exclude": ["node_modules"],
   "compilerOptions": {
-    "moduleResolution": "Node"
+    "moduleResolution": "Node10"
```
[the remark this answers is not in the document]
Comment (inpaceee) [reply packages/backend/tsconfig.json:6]: Yeah, https://www.typescriptlang.org/tsconfig/#moduleResolution

> 'node10' (previously called 'node') for Node.js versions older than v10, which only support CommonJS require. You probably won’t need to use node10 in modern code.

[the remark this answers is not in the document]
Comment (YohannesTesfay) [reply packages/backend/tsconfig.json:6]: Does that mean you don't need to use "Node10"?

[the remark this answers is not in the document]
Comment (inpaceee) [reply packages/backend/tsconfig.json:6]: I think it's fine to use on versions >10 (mine is 11.3.3 and it seems fine), but not the best. Docs suggest to use Node16 but that I think implies overhaul of our imports, not sure.

[the remark this answers is not in the document]
Comment (YohannesTesfay) [reply packages/backend/tsconfig.json:6]: I would then suggest `node16` or `nodenext`. 
Maybe @moncapitaine can also have a say.

**Comment by inpaceee** [comment]
Should be fixed in #965

**Pull request closed without merging**